CONTRIBUTING.md

```diff
@@ -0,0 +1 @@
+Contirbuting.md is available on the [wiki](https://github.com/musescore/MuseScore/wiki/Contributing.md)
```

Comment:
I've removed the `.md` from the end of the Wiki page so now it's just "Contributing".

Also, I think this wording is a bit clearer:

```markdown
If you would like to contribute, please see the [Contributing page](https://github.com/musescore/MuseScore/wiki/Contributing.md) on the GitHub Wiki. Thanks!
```

What do you think?

Reply:
Sounds good